src/components/CreateBoxe/CreateBoxe.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+function CreateBoxe() {
+  const [value, setValue] = useState('red');
+
+  function handleChange(event) {
+    setValue(event.target.value);
+  }
+
+  return (
+    <div>
+      <form action="" method="get" className="">
+        <div className="">
+          <label htmlFor="nameOfBoxes">Nom du coffret : </label>
+          <input type="text" name="name" id="name" required />
+        </div>
+        <div className="">
```

Comment:
maybe use a fieldset to group all bottle fields: https://developer.mozilla.org/fr/docs/Web/HTML/Element/fieldset